### PR TITLE
Adding support for cloudstack bundles

### PR DESCRIPTION
**Pull request by maxdrib**
*Issue #, if available:*

*Description of changes:*
This PR adds the CloudStack bundles to the list of bundles. After adding the types to the `release` directory, I ran `make manifests` to update the CRD's under `config/crd/bases/anywhere.eks.amazonaws.com_bundles.yaml`.

I have not yet been able to determine if kubeproxy is required or not

*Testing (if applicable):*
Successfully created CloudStack cluster when using this change (along with others)

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: release/api/v1alpha1/bundle_types.go [hunk 1]
```diff
@@ -196,6 +197,15 @@ type DockerBundle struct {
 	Metadata        Manifest `json:"metadata"`
 }
 
+type CloudStackBundle struct {
+	Version              string   `json:"version"`
+	ClusterAPIController Image    `json:"clusterAPIController"`
+	KubeProxy            Image    `json:"kubeProxy"`
+	Manager              Image    `json:"manager"`
```
Comment (abhay-krishna) [inline release/api/v1alpha1/bundle_types.go:204]: I think we don't need the `Manager` here. We only have the ClusterAPIController. For vSphere we had the `cloud-provider-vsphere` which was the cloud-controller-manager component.

File: release/pkg/assets_capc.go [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+	}
+
+	imageTagOverride := ImageTagOverride{
+		Repository: repoName,
+		ReleaseUri: imageArtifact.ReleaseImageURI,
+	}
+	imageTagOverrides = append(imageTagOverrides, imageTagOverride, kubeRbacProxyImageTagOverride)
+
+	manifestList := []string{
+		"infrastructure-components.yaml",
```
Comment (abhay-krishna) [inline release/pkg/assets_capc.go:85]: Don't we also need the cluster template manifest?

File: release/api/v1alpha1/bundle_types.go (hunk 1 above)
Comment (maxdrib) [reply release/api/v1alpha1/bundle_types.go:204]: Got it, will remove!

File: release/api/v1alpha1/bundle_types.go [hunk 3]
```diff
@@ -66,6 +66,7 @@ type VersionsBundle struct {
 	ControlPlane           KubeadmControlPlaneBundle   `json:"controlPlane"`
 	Aws                    AwsBundle                   `json:"aws"`
 	VSphere                VSphereBundle               `json:"vSphere"`
+	CloudStack             CloudStackBundle            `json:"cloudStack"`
```
Comment (g-gaston) [inline release/api/v1alpha1/bundle_types.go:69]:
```suggestion
	CloudStack             CloudStackBundle            `json:"cloudStack,omitempty"`
```
New fields should be optional to keep backwards compatibility :)

**Review by g-gaston — COMMENTED** [review]
One minor comment about json tags
I would leave up to @abhay-krishna to give the final `lgtm` since he is the release code expert 🥇

**Review by abhay-krishna — APPROVED** [review]
/lgtm

**Pull request merged**